### DISCUSSION TITLE
Nested query loop with filter data from parent result.

### DISCUSSION THREAD

**Discussion by tnchuntic** [WordPress/gutenberg#46643] [Ideas]
Asking if anyone have experience doing this scenario:

1. Nested query loop with filter taxonomy data from parent loop result.

ex. Product 1 single page have taxonomy 1 and we will use the taxonomy 1 to be a filter of the 2nd query loop. The goal of the nested query loop is to have a related products of the current product item displayed.

**Comment by carlomanf** [comment]
Yes, it's been raised as an issue before at https://github.com/WordPress/gutenberg/issues/31272 and https://github.com/WordPress/gutenberg/issues/39192
